SasquatchTV/SasquatchTVSwift/AppDelegate.swift

```diff
@@ -14,6 +14,11 @@ class AppDelegate : UIResponder, UIApplicationDelegate, CrashesDelegate {
 
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
+    // Set manual session tracker before start app center.
+    let sessionTrackerState = UserDefaults.standard.bool(forKey: kMSManualSessionTracker)
+  if (sessionTrackerState) {
```

Comment:
Formatting. And I think you can just `if UserDefaults.standard.bool(forKey: kMSManualSessionTracker) {`